src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/FallbackProjectManager.cs

```diff
@@ -44,7 +44,8 @@ internal sealed class FallbackProjectManager(
     {
         try
         {
-            if (_projectManagerAccessor.Instance.TryGetLoadedProject(razorProjectKey, out var project))
+            if (_projectManagerAccessor.TryGetInstance(out var projectSnapshotManager) &&
```

Comment:
Should this be moved outside this if, so that the `else` is not run either? The `AddFallbackProjectAsync` method would need the project manager, right? Or is it okay since it uses the dispatcher?

Reply:
No, I think we want the `else` block to run, create a fallback project, and add it to the project snapshot manager (creating it in the process). Not have a project snapshot manager instance yet is the same as "we haven't been told about the project yet".